static/js/message_fetch.js

```diff
@@ -292,13 +288,11 @@ export function load_messages(opts, attempt = 1) {
 }
 
 export function load_messages_for_narrow(opts) {
-    const msg_list = message_list.narrowed;
-
     load_messages({
         anchor: opts.anchor,
         num_before: consts.narrow_before,
         num_after: consts.narrow_after,
-        msg_list,
+        msg_list: opts.msg_list,
```

Comment:
I added this commit to pass the `msg_list` as a parameter, because it made for a more correct migration of this.